wordpress-mobile/WordPress-Android
# [Reader] Add "Tags" to dropdown menu

Fixes #20576

-----

## To Test:

- Install JP and sign in.
- Open "Reader".
- 🔍 Open the dropdown menu: there should be a "Tags" item with the correct icon (design: p1712144397494019/1712076563.723409-slack-C05N140C8H5) 
<img width="198" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/4f9306ec-cffd-4f0d-9434-caf318674c7a">

- 🔍 Tap the "Tags" item: an empty screen should be loaded (empty state and posts lists will be handled in separate tickets).
- 🔍 Open the other feed items: they should be working as expected.
- 🔍 Inspect the local DB: the "Tags" item only exists in memory, so it shouldn't be added to the local DB.
- 🔍 Create a new custom list in the web "Reader" and verify it's shown in the app to make sure the back-end integration is still working as expected.

-----

## Regression Notes

1. Potential unintended areas of impact

    - None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing.

3. What automated tests I added (or what prevented me from doing so)

    --

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
